crates/logfwd-io/codegen/opentelemetry-proto/v1.8.0/README.md

```
@@ -0,0 +1,13 @@
+Vendored OpenTelemetry protobuf definitions used by OTLP projection codegen.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)